test/unit/generator/content/schedule/part1/index.test.js

```diff
@@ -20,11 +20,11 @@ describe('schedule Part 1', () => {
 
```

Comment:
I think this test and all `index` tests should be under integration seeing as its result is from calling multiple units which we've tested individually.

What do you think?

Reply:
Missing checks for the outputs of `getSBI`, `getBusinessName` and `getAgreementReference`

OOS for this ticket but can you add a ticket to come back to add these.

Reply:
Checks for outputs are in the index test under integration folder.